<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@865.v43e78cc44e0d">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@3793.v65dec41c3a_c3">
    <parentIds>
      <string>42</string>
    </parentIds>
    <id>43</id>
    <startId>35</startId>
  </node>
  <actions>
    <wf.a.TimingAction plugin="workflow-api@1283.v99c10937efcb_">
      <startTime>1696445370221</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@1283.v99c10937efcb_">
      <error class="hudson.AbortException">
        <detailMessage>script returned exit code 1</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.handleExit(DurableTaskStep.java:668)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:614)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:555)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
          <trace>java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions>
          <wf.a.ErrorAction_-ErrorId>
            <stackTrace/>
            <suppressedExceptions class="empty-list"/>
            <uuid>27f11302-9bdc-4168-8964-f3e459d0174d</uuid>
          </wf.a.ErrorAction_-ErrorId>
        </suppressedExceptions>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>